docs/src/features.md

```diff
@@ -172,3 +172,51 @@ julia> summarize(s, ns=2)
  "Assume this Short Document as an example."
  "This has too foo sentences."
 ```
+
+##Parts of Speech Tagger
+
+This tagger can be used to find the POS tag of a word or token in a given sentence. It is a based on `Average Perceptron Algorithm`.
+The model can be trained from scratch and weights are saved in specified location.
+The pretrained model can also be loaded and can be used directly to predict tags.
+
+#To train model:
```

Comment:
Can you remove this as a main heading of the document? I think it is better as plain text explaining with a sentence or 2, or a sub-sub heading (using `### `).

Check out the julia docs - https://docs.julialang.org/en/v1/stdlib/Markdown/#Headers-1